### PR TITLE
add music album and genre class tables

### LINKED ISSUE #14: [2pt #2] Create a schema.sql
Create a schema.sql file with tables that will be analogical to the structure of the classes that you created:

- music_albums table (add all properties and associations from the parent Item class as table columns)
- genres table

### PR DESCRIPTION
**Pull request by Balikuddembe**
In this PR,
* Create a schema.sql file with tables that will be analogical to the structure of the classes that you created:
    *  music_albums table (add all properties and associations from the parent Item class as table columns)
    genres table

### CODE REVIEW

File: src/database/schema.sql
Comment (lucascomamala) [inline src/database/schema.sql]: As per the instructions, the child class tables should have ALL of the properties (label, author, genre) from the parent Item table, which then reference the association tables. The item_id  attribute is not necessary

File: src/database/schema.sql
Comment (lucascomamala) [inline src/database/schema.sql]: The item_id property is not needed here.

**Review by lucascomamala — CHANGES_REQUESTED** [review]
Hi @Balikuddembe , 

Good job so far!
There are some issues that need to be addressed before approval

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

File: src/database/schema.sql
Comment (Balikuddembe) [reply src/database/schema.sql]: Thaank you @lucascomamala for this review comment.Let me implement it

File: src/database/schema.sql
Comment (lucascomamala) [inline src/database/schema.sql:55]: Great job removing the unnecessary attributes but you still need to include columns for ALL of the properties inherited from the Item class (genre, author, label). Take a look at how the game and book tables are modeled for a clue.

**Review by lucascomamala — CHANGES_REQUESTED** [review]
Hi @Balikuddembe , 

Good job so far!
There are some issues that need to be addressed before approval

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

**Review by lucascomamala — APPROVED** [review]
Hi @Balikuddembe , 

# ✔️   Status: APPROVED 🎉 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 

## Top highlights 🎯

- All requirements complete. ✅ 
- Good SQL code. ✅

**Pull request merged**
